kernel/sched.c

```diff
@@ -1237,14 +1238,18 @@ static inline void z_vrfy_k_yield(void)
 #include <syscalls/k_yield_mrsh.c>
 #endif
 
-static int32_t z_tick_sleep(int32_t ticks)
+static int32_t z_tick_sleep(k_ticks_t ticks)
 {
 #ifdef CONFIG_MULTITHREADING
 	uint32_t expected_wakeup_time;
 
 	__ASSERT(!arch_is_in_isr(), "");
 
-	LOG_DBG("thread %p for %d ticks", _current, ticks);
+#ifdef CONFIG_TIMEOUT_64BIT
+	LOG_DBG("thread %p for %" PRId64 " ticks", _current, ticks);
```

Comment:
I forgot that the logger API doesn't support 64bit values regardless of the format string.
The xtensa platform should probably be added back in, and this print excluded for 64bit?
Or perhaps print out the value mod 2**31 here.

Reply:
Sorry the whole logging subsystem does not support 64 bit or the problem is xtensa specific ? AFAIU the problem is in xtensa SDK, PRI*64 is not being properly defined there. If that is the case I think is fine just exclude the platform from the test. This problem is being addressed in the next SDK release AFAIK.

Reply:
The logging API doesn't support 64-bit values period.  https://github.com/zephyrproject-rtos/zephyr/issues/18351#issuecomment-522276594

Reply:
Thanks ! Just logging it conditionally now.